library/std/src/process.rs

```diff
@@ -2320,8 +2320,8 @@ pub fn exit(code: i32) -> ! {
 /// The function will never return and will immediately terminate the current
 /// process in a platform specific "abnormal" manner.
 ///
-/// Note that because this function never returns, and that it terminates the
-/// process, no destructors on the current stack or any other thread's stack
+/// Note that because this function never returns and terminates the process,
```

Comment:
Hmm I'm not sure this is better, it could be confused for "this function never returns and never terminates the process". Maybe flipping it around to "this function terminated the process and never returns" would be better?

Reply:
Good point! And seeing this here it seems there is some duplication of the sentence above anyway, which is clear because of using "will", so I may try to fuse hose together.